containers/ecr-viewer/integration/core_database_repo.test.ts

```diff
@@ -70,68 +81,137 @@ describe("database_repo", () => {
         date_created: new Date("2025-01-01"),
         report_date: new Date("2025-02-07"),
       });
+      const actual = await database_repo.findEcrById("54321");
+      expect(actual).toEqual({
+        eICR_ID: "54321",
+        set_id: "setid",
+        data_source: "DB",
+        fhir_reference_link: "link",
+        eicr_version_number: "50000",
+        patient_name_first: "Boba",
+        patient_name_last: "Fett",
+        patient_birth_date: new Date("1969-02-10T05:00:00.000Z"),
+        date_created: new Date("2025-01-01"),
+        report_date: new Date("2025-02-06T05:00:00.000Z"),
+      });
     });
 
     it("should delete an ECR with a given id", async () => {
       await database_repo.deleteEcr("12345");
+      const actual = await database_repo.findEcrById("12345");
+      expect(actual).toBeUndefined();
     });
   });
 
   // ecr_rr_conditions
   describe("ecr_rr_conditions", () => {
+    var expected = {
+      eICR_ID: "12345",
+      uuid: "12345",
+      condition: "Dark Magic",
+    };
+    beforeEach(async () => {
+      await database_repo.createEcrCondition(expected);
+    });
+
+    afterEach(async () => {
+      await clearCore();
+    });
+
     it("should find a conditions with a given uuid", async () => {
-      await database_repo.findEcrConditionById("12345");
+      const actual = await database_repo.findEcrConditionById("12345");
+      expect(actual).toEqual(expected);
     });
 
     it("should find all conditions named Dark Magic", async () => {
-      await database_repo.findEcrCondition({ condition: "Dark Magic" });
+      const actual = await database_repo.findEcrCondition({
+        condition: "Dark Magic",
+      });
+      expect(actual[0]).toEqual(expected);
     });
 
     it("should update the condition with a given id", async () => {
       await database_repo.updateEcrCondition("12345", {
         condition: "Extra Dark Magic",
       });
+      const actual = await database_repo.findEcrConditionById("12345");
+      expect(actual?.condition).toEqual("Extra Dark Magic");
     });
 
     it("should create a condition", async () => {
       await database_repo.createEcrCondition({
-        eICR_ID: "12345",
-        uuid: "setid",
+        eICR_ID: "54321",
+        uuid: "54321",
+        condition: "Dark Magic",
+      });
+      const actual = await database_repo.findEcrConditionById("54321");
+      expect(actual).toEqual({
+        eICR_ID: "54321",
+        uuid: "54321",
         condition: "Dark Magic",
       });
     });
 
     it("should delete a condition with a given id", async () => {
       await database_repo.deleteEcrCondition("12345");
+      const actual = await database_repo.findEcrConditionById("12345");
+      expect(actual).toBeUndefined();
     });
   });
 
   // ecr_rr_rule_summaries
   describe("ecr_rr_rule_summaries", () => {
+    const expected = {
+      uuid: "12345",
+      ecr_rr_conditions_id: "12345",
```

Comment:
```suggestion
      uuid: "12345",
      ecr_rr_conditions_id: "123",
```
can we use different IDs here?